pkgs/by-name/al/allmark/package.nix

```diff
@@ -15,11 +16,26 @@ buildGoModule rec {
     hash = "sha256-JfNn/e+cSq1pkeXs7A2dMsyhwOnh7x2bwm6dv6NOjLU=";
   };
 
-  postPatch = ''
-    go mod init github.com/andreaskoch/allmark
-  '';
+  vendorHash = "sha256-32MDPl+8lfUqyhG0rjZhpGMJzgIvb2E0xOTQ6wDKJto=";
+
+  deleteVendor = true;
 
-  vendorHash = null;
+  patches = [
+    # add go module
+    (fetchpatch {
```

Comment:
Should vendor them here, see https://github.com/NixOS/nixpkgs/tree/master/pkgs#patches